activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -43,9 +43,11 @@ def set_fixture_class(class_names = {})
       def fixtures(*fixture_set_names)
         if fixture_set_names.first == :all
           raise StandardError, "No fixture path found. Please set `#{self}.fixture_path`." if fixture_path.blank?
-          fixture_set_names = Dir[::File.join(fixture_path, "{**,*}/*.{yml}")].uniq
-          fixture_set_names.reject! { |f| f.start_with?(file_fixture_path.to_s) } if defined?(file_fixture_path) && file_fixture_path
-          fixture_set_names.map! { |f| f[fixture_path.to_s.size..-5].delete_prefix("/") }
+          fixture_set_names = Array(fixture_path).flat_map do |path|
+            names = Dir[::File.join(path, "{**,*}/*.{yml}")].uniq
+            names.reject! { |f| f.start_with?(file_fixture_path.to_s) } if defined?(file_fixture_path) && file_fixture_path
+            names.map! { |f| f[path.to_s.size..-5].delete_prefix("/") }
+          end.uniq
```

Comment:
The loop here might be a problem from a performance standpoint. We could join all the paths together like I do in `fixtures.rb` above - but then I am not sure how to do line 49. Line 49 is removing the front of the path, but NOT the namespace. Eg `myapp/test/fixtures/orders/line_items.yml` becomes `orders/line_items`. I paired on this with Gannon but we couldn't think of a better way.